cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
Well this case may not be generalized enough :) but we use cgo and with the
latest Go you can't statically link with cgo unless you specify the
compiler and do a few more things. Doing those seemed like a huge pain so
we opted not to. We needed a busybox anyways so we could `docker exec` and
inspect the environment.

On Fri, May 15, 2015 at 3:37 PM, Tim Hockin <notifications@github.com>
wrote:

> In cluster/addons/dns/kube2sky/Dockerfile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322#discussion_r30451593>
> :
>
> > @@ -1,4 +1,4 @@
> > -FROM scratch
>
> Eh? Explain?
> … <#14d59b9677004f37_>
> On Fri, May 15, 2015 at 3:33 PM, Victor Marmol <notifications@github.com>
> wrote: In cluster/addons/dns/kube2sky/Dockerfile <
> https://github.com/GoogleCloudPlatform/kubernetes/pull/8322#discussion_r30451448>
> : > @@ -1,4 +1,4 @@ > -FROM scratch You can't actually statically link some
> dynamic libs dependencies :) we stopped being able to statically link
> cAdvisor for example. — Reply to this email directly or view it on GitHub <
> https://github.com/GoogleCloudPlatform/kubernetes/pull/8322/files#r30451448>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322/files#r30451593>
> .
>